genesyscloud/telephony_providers_edges_site/resource_genesyscloud_telephony_providers_edges_site.go

```diff
@@ -0,0 +1,358 @@
+package telephony_providers_edges_site
```

Comment:
Basically everything I said about the phones resource file and utils file applies here. Try using a `getXFromResourceData` function, setting values on objects directly instead of using variables when possible and using the functions in the `resourcedata` package to reduce unnecessary code